client/my-sites/woocommerce/woop/landing-page.tsx

```diff
@@ -65,20 +59,27 @@ const WoopLandingPage: React.FunctionComponent< Props > = ( { siteId } ) => {
 		<div className="woop__landing-page">
 			<FixedNavigationHeader navigationItems={ navigationItems } contentRef={ ctaRef }>
 				<Button onClick={ onCTAClickHandler } primary disabled={ isTransferringBlocked }>
-					{ __( 'Set up my store!' ) }
+					{ __( 'Start a new store' ) }
 				</Button>
 			</FixedNavigationHeader>
-			<CtaSection
-				title={ __( 'Have something to sell?' ) }
-				headline={ __( 'Build exactly the eCommerce website you want.' ) }
-				buttonText={ __( 'Set up my store!' ) }
-				buttonAction={ onCTAClickHandler }
-				buttonDisabled={ isTransferringBlocked }
-				ctaRef={ ctaRef }
-				notice={ renderWarningNotice() }
-			>
-				<MasonryWave images={ images } />
-			</CtaSection>
+			{ renderWarningNotice() }
+			<EmptyContent
+				title={ __( 'Set up a store and start selling online' ) }
```

Comment:
I changed this to "Set up" rather than "Setup" because we say "Set up a new store..." in the subheader. And setup is a noun. 🙂